augur/cli/db.py

```diff
@@ -81,6 +82,47 @@ def add_repo_groups(augur_app, filename):
             else:
                 logger.info(f"Repo group with ID {row[1]} for repo group {row[1]} already exists, skipping...")
 
+@cli.command('add-github-org')
+@click.argument('organization_name')
+@pass_application
+def add_github_org(augur_app, organization_name):
+    """
+    Create new repo groups in Augur's database
+    """
+    org_query_response = requests.get(f"https://api.github.com/orgs/{organization_name}").json()
+    if "login" in org_query_response:
+        logger.info(f"Organization \"{organization_name}\" found")
+    else:
+        logger.fatal(f"No organization with name {organization_name} could be found")
+        exit(1)
+
+    page = 1
+    headers = {'Authorization': 'token %s' % augur_app.config.get_value("Database", "key")}
+    all_repos = []
+    repo_query_response = requests.get(org_query_response['repos_url'] + f"?per_page=100&page={page}", headers=headers).json()
+    while repo_query_response != []:
+        for repo in repo_query_response:
+            all_repos.append(repo)
+
+        page+=1
+        repo_query_response = requests.get(org_query_response['repos_url'] + f"?per_page=100&page={page}", headers=headers).json()
+
+    insert_repo_group_sql = s.sql.text("""
+    INSERT INTO "augur_data"."repo_groups"("rg_name", "rg_description", "rg_website", "rg_recache", "rg_last_modified", "rg_type", "tool_source", "tool_version", "data_source", "data_collection_date") VALUES (:repo_group_name, '', '', 0, CURRENT_TIMESTAMP, 'Unknown', 'Loaded by user', '1.0', 'Git', CURRENT_TIMESTAMP) RETURNING repo_group_id;
+    """)
+    new_repo_group_id = augur_app.database.execute(insert_repo_group_sql, repo_group_name=organization_name).fetchone()[0]
+
+    insert_repo_sql = s.sql.text("""
+        INSERT INTO augur_data.repo(repo_group_id, repo_git, repo_status,
+        tool_source, tool_version, data_source, data_collection_date)
+        VALUES (:repo_group_id, :repo_git, 'New', 'CLI', 1.0, 'Git', CURRENT_TIMESTAMP)
+    """)
+    logger.info(f"{organization_name} repo group created")
+
+    for repo in all_repos:
+        logger.info(f"Adding {organization_name}/{repo['name']} ({repo['clone_url']})")
+        result = augur_app.database.execute(insert_repo_sql, repo_group_id=new_repo_group_id, repo_git=repo['clone_url'])
```

Comment:
There is a way to bulk insert which is faster, but if performance is not an issue (which I don't imagine it would be because I wouldn't think there would be an insane amount of repos in any org) then this implementation is just fine. So lmk if you want me to point you to some resources

Reply:
I considered a bulk insert but since there won't be too many repos for the vast majority of cases I went with this